cds-snc/report-a-cybercrime
# Fix incorrect self harm alarm warning on server while no self harm is detected

Fixes #2333 (issue)

# Description
This is to fix self harm alarm warning incorrectly logged on server when no self harm is detected

> Please include a summary of the change.

# Any new packages installed?

> Give details
How to test:
1) Use harmful words in text description inputs, and submit report, check server side log, you should see something like:
"Self harm words detected: suicide,kill,knife,depressed" 

2) Do not use any harmful words in text description inputs, and submit report, check server side log,  you should not see any thing like
"Self harm words detected"

# Required followup work

> Is there anything related to this that still needs to be done (ex: documentation changes).

# Checklist:

- [ ] I have updated the azurescript.sh with any **new environment variables** and added them to the appsettings
- [X ] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [ ] I have added documentation to any modified front-end code. (Or added missing documentation)
